src/moscot/base/problems/birth_death.py

```diff
@@ -183,6 +184,8 @@ def estimate_marginals(
             Key in :attr:`~anndata.AnnData.obs` where proliferation scores are stored.
         apoptosis_key
             Key in :attr:`~anndata.AnnData.obs` where apoptosis scores are stored.
+        scaling
```

Comment:
Can you please elabore a little bit on what this means? Not as long as in the corresponding example, but the docstrings here are fairly short, so we can spend a little space on this.

Reply:
I.e. specify what happens if it's `None`, and if it's `float`